WordPress/src/main/res/values/dimens.xml

```diff
@@ -434,7 +434,6 @@
     <dimen name="domain_suggestions_intro_horizontal_padding">16dp</dimen>
     <dimen name="domain_suggestions_intro_header_bottom_padding">8dp</dimen>
 
-    <dimen name="keyline_first">16dp</dimen>
     <dimen name="keyline_second">72dp</dimen>
```

Comment:
Not directly related to this PR, so no action required, but just to share this thought 😊; it sounded strange that we are not using anywhere the `keyline_first`. Probably we ended up using other `16dp` constant instead of this explicit one.